templates/zerver/help/mute-a-user.md

```diff
@@ -0,0 +1,75 @@
+# Mute a user
+
+!!! tip ""
+    This feature works on an individual basis. In case of a user seriously
+    violating the code of conduct of your community, consider requesting
+    your organization administrator to [deactivate](/help/deactivate-or-reactivate-a-user)
+    them instead.
+
+
+To deal with annoying or troublesome users, you can mute them.
+A user you mute will not know that they have been muted, and muting
+someone will have the following effects:
+
+- Stream messages sent by muted users will be hidden under a
+  **Click to reveal** banner, and [private messages](/help/private-messages)
+  will be completely excluded from the message view.
+
+- Messages sent by a muted user will automatically be
+  [marked as read](/help/marking-messages-as-read) for you, and
+  will never generate any desktop, push or email notifications.
+
+- Muted users will be excluded from the right sidebar, the
+  [private conversations](/help/private-messages) list, and from
+  the [mentions](/help/mention-a-user-or-group) autocomplete.
+
+- Muting a user does not affect their ability to send you messages
+  or view your [status and availability](/help/status-and-availability).
+
+
+### From the message view
+
+{start_tabs}
+
+1. Click on a user's profile picture or [mention](/help/mention-a-user-or-group).
+
+1. Click on **Mute this user**
+
+1. On the confirmation popup, click on **Confirm**
+
+{end_tabs}
+
+Follow the same procedure to unmute the user.
+
+### Via the right sidebar
+
+{start_tabs}
+
+1. Hover over a user's name in the right sidebar.
+
+1. Click on the ellipsis (<i class="zulip-icon zulip-icon-ellipsis-v-solid"></i>) to
+  the right of their name.
+
+1. Click on **Mute this user**
+
+1. On the confirmation popup, click on **Confirm**
```

Comment:
These numbered list entries should be consistent about whether the end with a `.` with other /help/ pages.

Reply:
Fixed.